source/crud/tests/read/aggregate-collation.yml

```diff
@@ -0,0 +1,18 @@
+data:
+    - {_id: 1, x: 'ping'}
+minServerVersion: '3.4'
+maxServerVersion:
```

Comment:
Since both fields are optional, I think we can simply omit the key when there is no value.